tachyon/__main__.py

```diff
@@ -251,6 +262,10 @@ def main(*, target_host, cookie_file, json_output, max_retry_count, plugin_setti
     # Set conf values
     conf.target_host = parsed_url.netloc
     conf.base_url = "%s://%s" % (parsed_url.scheme, parsed_url.netloc)
+    if pre_crawled_paths:
+        conf.pre_crawled_paths = pre_crawled_paths.split(",")
```

Comment:
String splitting can be avoided by using the click multiple options.

https://click.palletsprojects.com/en/8.0.x/options/#multiple-options

Reply:
Minor point, but the multiple option would require repeating the flag as well, e.g. `--pre-crawled-paths /test1 --pre-crawled-paths /test2`. I can either leave as-is or add a short flag like `-p /test1 -p /test2` and remove the splitting.